DFC.ServiceTaxonomy.Neo4j/TestingVisibility.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("DFC.ServiceTaxonomy.UnitTests")]
+[assembly: InternalsVisibleTo("DFC.ServiceTaxonomy.IntegrationTests")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
+
+//todo: move core of validation into a validation service in the neo project
+// but for now...
+[assembly: InternalsVisibleTo("DFC.ServiceTaxonomy.GraphSync")]
```

Comment:
Would you mind changing the name of this file, as the last `InternalsVisibleTo` is functional, rather than test related.